src/sdk/formatting/formatTimeRange.test.ts

```diff
@@ -1,24 +1,28 @@
 import { formatTimeRange } from './formatTimeRange';
 import expect = require('expect.js');
 
+const timeZoneCode = ' UTC-7';
```

Comment:
Should this be factored out into a single file that is imported into each test file that uses it?

Reply:
Wasn't sure what to name that file. I called it `testUtil.ts`. Happy to rename it if needed